src/youtube.js

```diff
@@ -8,15 +8,15 @@
 function stripParameters(shortcodeString) {
 	// Split parameters or split folder separator
 	if (shortcodeString.includes('?')) {
-		return shortcodeString.split('?')[0];
+		shortcodeString = shortcodeString.split('?')[0];
 	}
 
 	if (shortcodeString.includes('/')) {
-		return shortcodeString.split('/')[0];
+		shortcodeString = shortcodeString.split('/')[0];
 	}
 
 	if (shortcodeString.includes('&')) {
-		return shortcodeString.split('&')[0];
+		shortcodeString = shortcodeString.split('&')[0];
```

Comment:
Curious: why did you make these changes? They seem to be unrelated to the PR, plus I prefer to let the function return as soon as it finds what it needs, so that it doesn't still have to do all these unnecessary checks.